docs/en/tutorials/how_to.md

```diff
@@ -0,0 +1,60 @@
+# Tutorial 11: Collection of common usage
+
+This tutorial mainly collects common usages in MMDetection
+
+## Backbone network using MMClassification
+
+The Hierarchy Registry function proposed in MMCV allows modules in various algorithm libraries of OpenMMLab to call each other. Users can use backbone networks from MMClassification in MMDetection without implementing a network that already exists in MMClassification for MMDetection as well.
+
+Suppose you want to use `MobileNetV3-small` as the backbone network of `RetinaNet`, the sample code is as follows.
+
+```python
+_base_ = [
+    '../_base_/models/retinanet_r50_fpn.py',
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+# please install mmcls>=0.20.0
+# import mmcls.models to trigger register_module in mmcls
+custom_imports = dict(imports=['mmcls.models'], allow_failed_imports=False)
+pretrained = 'https://download.openmmlab.com/mmclassification/v0/mobilenet_v3/convert/mobilenet_v3_small-8427ecf0.pth'
+model = dict(
+    backbone=dict(
+        _delete_=True,
+        type='mmcls.MobileNetV3', # Using MobileNetV3 from mmcls
```

Comment:
May explain the key is to specify `mmcls.xxx`, and explain _delete_. We should also tell users to refer to the documentation of mmcls about how to write the config of backbone.